tests/test_wave_01.py

```diff
@@ -119,12 +119,13 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     assert len(updated_data["watchlist"]) == 0
     assert len(updated_data["watched"]) == 1
     
-    raise Exception("Test needs to be completed.")
+    #raise Exception("Test needs to be completed.")
+    assert janes_data["watched"][-1]["title"] == MOVIE_TITLE_1
```

Comment:
This assert isn't testing the right data structure; `watch_movie` may have side effects, but the return value is what we're testing here.  Additionally, tests work better when they are comprehensive, so in this case I recommend checking all of the keys of the dictionary. Checking the entire data structure ensures that the function being tested here didn't corrupt any of the data.
```suggestion
    assert updated_data["watched"][0]["title"] == MOVIE_TITLE_1
    assert updated_data["watched"][0]["genre"] == GENRE_1
    assert updated_data["watched"][0]["rating"] == RATING_1
```